strax/processing/peak_building.py

```diff
@@ -157,10 +164,10 @@ def sum_waveform(peaks, records, adc_to_pe, select_peaks_indices=None):
         return
     dt = records[0]['dt']
 
-    # Big buffer to hold even largest sum waveforms
-    # Need a little more even for downsampling..
```

Comment:
Specific reason to remove the comment? If still true, could you please add it again?